### ISSUE TITLE
bug: Icon is not accessible with voiceover navigating using keyboard

### ISSUE THREAD

**Issue by Malnag** [ionic-team/ionic-framework#27764]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

v7.x

### Current Behavior

On the login page with user I’d and password fields 

user can access the show password icon only when they navigating the page from top to bottom using gestures. However icon is not accessible when a user navigates to password field using the next button on keyboard in ionic 7

### Expected Behavior

Access the show icon which is next element in password field when next is selected

### Steps to Reproduce

Login form with userid and password
Inside password field there is show password icon
When the user select next on the keyboard it has to focus the show password icon 

### Ionic Info

Version 7

**Comment by amandaejohnston** [comment]
This issue is being closed due to lack of a reproduction. If this is still a problem in the latest version of Ionic, please open a new issue and ensure the template is fully filled out. Thank you!
